win/rl/pynethack.cc

```diff
@@ -392,19 +420,20 @@ PYBIND11_MODULE(_pynethack, m)
            [](int glyph) { return glyph_is_warning(glyph); });
 
     py::class_<permonst>(mn, "permonst", "The permonst struct.")
-        .def("__init__",
-             // See https://github.com/pybind/pybind11/issues/2394
-             [](py::detail::value_and_holder &v_h, int index) {
-                 if (index < 0 || index >= NUMMONS)
-                     throw std::out_of_range(
-                         "Index should be between 0 and NUMMONS ("
-                         + std::to_string(NUMMONS) + ") but got "
-                         + std::to_string(index));
-                 v_h.value_ptr() = &mons[index];
-                 v_h.inst->owned = false;
-                 v_h.set_holder_constructed(true);
-             },
-             py::detail::is_new_style_constructor())
+        .def(
```

Comment:
I have no idea why clang-format decided to change this.

I wish it was more stable version by version.

Reply:
Just a heads up that you should install clang-format in your conda env (`conda install clang-format`) to get a recent version, since I've noticed system package managers seem to have very old versions.